PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureRunProfileState.java

```diff
@@ -46,26 +52,41 @@ public ExecutionResult execute(Executor executor, @NotNull ProgramRunner program
         ConsoleView consoleView = TextConsoleBuilderFactory.getInstance().createBuilder(this.project).getConsole();
         processHandler.startNotify();
         consoleView.attachToProcess(processHandler);
+
+        Hooks.onErrorDropped(ex -> {
+            if (Exceptions.getFinalCause(ex) instanceof InterruptedException) {
+                LOGGER.info(ex.getMessage());
+            }
+            throw Lombok.sneakyThrow(ex);
+        });
+
         final Operation operation = createOperation();
-        Observable.fromCallable(
-            () -> {
-                try {
-                    operation.start();
-                    return this.executeSteps(processHandler, operation);
-                } finally {
-                    // Once the operation done, whether success or not, `setText` should not throw new exception
-                    processHandler.setProcessTerminatedHandler(RunProcessHandler.DO_NOTHING);
-                }
-            }).subscribeOn(SchedulerProviderFactory.getInstance().getSchedulerProvider().io()).subscribe(
-                (res) -> {
-                    this.sendTelemetry(operation, null);
-                    this.onSuccess(res, processHandler);
-                },
-                (err) -> {
-                    err.printStackTrace();
-                    this.sendTelemetry(operation, err);
-                    this.onFail(err, processHandler);
-                });
+        final Disposable subscribe = Mono.fromCallable(() -> {
```

Comment:
rollback to `Observable` since `Mono` has no exception handler for now.

Reply:
onFail has handled the exception